### PR TITLE
fix: fix release get hosts

### PR DESCRIPTION
**Pull request by Han-Ya-Jun**
### Description
修复发布相关问题(open api 添加lock and fix release get hosts)

Fixes # (issue)

### Checklist

- [x] 填写 PR 描述及相关 issue (write PR description and related issue)
- [x] 代码风格检查通过 (code style check passed)
- [ ] PR 中包含单元测试 (include unit test)
- [x] 单元测试通过 (unit test passed)
- [x] 本地开发联调环境验证通过 (local development environment verification passed)

### CODE REVIEW

**Review by wklken — APPROVED** [review]
LGTM

File: src/dashboard/apigateway/apigateway/core/models.py [hunk 1]
```diff
@@ -519,7 +519,7 @@ def object_display(self):
         if not self.version:
             return f"{self.name}({self.title})"
 
-        return f"{self.version}({self.title})"
+        return f"{self.version}"
```
Comment (alex-smile) [inline src/dashboard/apigateway/apigateway/core/models.py:522]: self.version 本身就是一个字符串，不需要 f"" 格式化

File: src/dashboard/apigateway/apigateway/controller/crds/v1beta1/convertors/resource.py [hunk 2]
```diff
@@ -178,7 +178,7 @@ def _convert_http_resource_upstream(self, resource_proxy: Dict[str, Any], backen
         )
 
         for i in upstreams.get("hosts", []):
```
Comment (alex-smile) [inline src/dashboard/apigateway/apigateway/controller/crds/v1beta1/convertors/resource.py:180]: 要不要把 i 改成 host？

File: src/dashboard/apigateway/apigateway/controller/crds/release_data/release_data.py [hunk 3]
```diff
@@ -149,6 +149,10 @@ def _resources_plugins(self) -> Dict[int, List[PluginData]]:
     def get_resources_upstream(self, resource_proxy: Dict[str, Any], backend_id: int):
         return resource_proxy.get("upstreams")
 
+    def get_upstream_host(self, upstream: Dict[str, Any]) -> str:
+
```
Comment (alex-smile) [inline src/dashboard/apigateway/apigateway/controller/crds/release_data/release_data.py:153]: 去除这个空行

**Review by alex-smile — APPROVED** [review]

File: src/dashboard/apigateway/apigateway/core/models.py (hunk 1 above)
Comment (Han-Ya-Jun) [reply src/dashboard/apigateway/apigateway/core/models.py:522]: done

File: src/dashboard/apigateway/apigateway/controller/crds/v1beta1/convertors/resource.py (hunk 2 above)
Comment (Han-Ya-Jun) [reply src/dashboard/apigateway/apigateway/controller/crds/v1beta1/convertors/resource.py:180]: done

**Pull request merged**